spec/01_calculator_spec.rb

```diff
@@ -0,0 +1,32 @@
+# frozen_string_literal: true
+
+# Specification for RubyMonk Primer Problem 1 Calculator class
+require 'faker'
+require_relative '../01_calculator'
+
+RSpec.describe Calculator do
+  subject(:calculator) { described_class.new }
+
+  describe '#add' do
+    it 'adding 2 and 3 gives 5' do
```

Comment:
Use `specify`, and not `it`.  Or `context` if you prefer it.  Either will read a bit more like English.

Reply:
I'd slightly disagree on this, though I do not use `specify` so I am unfamiliar with it. Generally, I'd use `context` to break up...well, context. These examples are a bit too simple for it, but it would be something like

```ruby
context 'when adding two positive numbers' do
  it 'correctly performs the addition' do
    ...
  end
end

context 'when adding two negative numbers' do
  it 'correctly performs the addition' do
    ...
  end
end

context 'when adding a negative number and a positive number' do
  it 'correctly performs the addition' do
    ...
  end
end
```

Reply:
Specify would work as so: `specify 'adding 2 and 3 gives 5'`.  It just reads a bit more English-y.

I prefer `context` as well, but also agree these examples might be too simple for it.

Reply:
Yeah, I went with the more simpler style which matched more of the original tests.  I actually went the full context route in `16_my_array_spec.rb` covering all the conditional combinations with passing a block and initial value and then rubocop-rspec complained about too many levels of context, so I had to refactor.  

Personally `specify` usually doesn't read better to me and I like the brevity and certainty of `it` in most cases.  But I will consider using `specify` more.

Reply:
Then, `it 'gives 5 when provided 2 and 3'` is also acceptable to me.  My concern was purely semantic -- erm, syntactic, technically.

#linguist

Reply:
✅